### ISSUE TITLE
nothing works

### ISSUE THREAD

**Issue by 3Dietrich** [Maxattax97/openscad-format#14]
in VSCode I found "openscad-format" v1.0.1 as a possible openscad formater. Installed and.. an error (on both Catalina and BigSure):
"**The 'openscad-format' command is not available. Please check your openscad-format.executable user setting and ensure it is installed.**"

I don't exactly know what it expects from me, both settings seem to be normal
<img width="257" alt="Bildschirmfoto 2021-05-12 um 12 34 01" src="https://user-images.githubusercontent.com/4608796/117961426-646a2200-b31e-11eb-90c6-2ababb0a0242.png">
ok, I tried to (re)install it by following requirements  [openscad-format](https://github.com/Maxattax97/openscad-format)
but I have no "npm" installed  - so I pressed the link for [npm](https://www.npmjs.com/package/openscad-format) - what showed almost the same page.. wtf
Watching https://www.npmjs.com/ (what offers npm?) doesn't bring me to a result, it's about JavaScript and I need to sign up to get anything.. common.. I wanted to install an openscad formatter, not bying an abo for a tool for a tool for installing the formatter!
Since this formatter seems the only openscad format I'm bound to it, or I have bad luck. 

long story short:
What is the reason that it is installed but not available?
